doc-surrealdb_versioned_docs/version-2.x/surrealql/statements/define/database.mdx

```diff
@@ -37,4 +37,83 @@ The `IF NOT EXISTS` clause can be used to define a database only if it does not
 ```surql
 -- Create a DATABASE if it does not already exist
 DEFINE DATABASE IF NOT EXISTS app_vitalsense;
+```
+
+## Changefeeds
+
+A changefeed can be defined on a database, in which case the changefeed will track changes made to every table in the database.
```

Comment:
@Dhghomon  Since a `CHANGEFEED` is defined on a table is implicitly created on the database. I wonder if this is redundant.

P.S: This is a 2.0 file.